src/index.html

```diff
@@ -6,14 +6,483 @@
+            <a
+              href="#contact-form"
+              class="button"
+            >
+              Apply
+            </a>
+          </div>
+        </div>
+      </section>
+    </main>
+
+    <footer class="footer">
+      <div class="container">
+        <div class="footer__wrapper">
+          <form
+            class="contact-form"
+            id="contact-form"
+          >
+            <label
+              class="contact-form__label"
+              for="contact-form"
+            >
+              Send us a message
+            </label>
+            <input
+              type="email"
+              name="email"
+              placeholder="Your email"
+              class="contact-form__input"
+              required
+            />
+            <input
+              type="text"
+              name="name"
+              placeholder="Your name"
+              class="contact-form__input"
+              required
+            />
+            <textarea
+              type="text"
+              name="message"
+              placeholder="Your message"
+              class="contact-form__input contact-form__input--message"
+              required
+            ></textarea>
+            <button
+              type="submit"
+              class="button contact-form__button"
+            >
+              Send
+            </button>
+          </form>
+
+          <address
+            class="contacts"
+            id="contacts"
+          >
+            <h2 class="contacts__title">Contact us</h2>
+            <div class="contacts__wrapper">
+              <label class="contacts__item">
+                call us
+                <a
+                  href="tel:+654 321 987"
+                  class="contacts__link"
+                >
+                  654 321 987
+                </a>
+              </label>
+
+              <label class="contacts__item">
+                visit us
+                <a
+                  href="https://www.google.com/maps/@42.169758,-88.0347239,12z?entry=ttu"
+                  target="_blank"
+                  class="contacts__link"
+                >
+                  2905 West Drive, Buffalo Grove
+                </a>
+              </label>
+
+              <label class="contacts__item">
+                our socials
+                <div class="contacts__social-wrapper">
+                  <a
+                    href="http://facebook.com"
+                    target="_blank"
+                    class="contacts__social contacts__social--facebook"
+                  ></a>
+
+                  <a
+                    href="http://twitter.com"
+                    target="_blank"
+                    class="contacts__social contacts__social--twitter"
+                  ></a>
+
+                  <a
+                    href="http://instagram.com"
+                    target="_blank"
+                    class="contacts__social contacts__social--instagram"
+                  ></a>
```

Comment:
Always highlight clickable elements when hovered to inform user an interaction is possible [Reference](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules#:~:text=Hightlight%20clickable%20element%20with%20cursor%3A%20pointer%20and%20some%20%3Ahover%20styles)